rero/rero-ils
# ui: add switch to professional view

* NEW Adds 'switch to professional views' link.
* BETTER Removes useless template and config.

Co-Authored-by: Alicia <alicia.zangger@rero.ch>

## Why are you opening this PR?

To implement task 1199 of US987

## How to test?

* Login as librarian or system librarian and check the menu. The link should not be available for patrons.
* Test the link: should go to home page of professional ui.

## Screenshots
<img width="359" alt="Screen Shot 2019-12-10 at 15 08 37" src="https://user-images.githubusercontent.com/28982829/70536482-1405f800-1b5f-11ea-8c13-77ad9da922df.png">

## Code review check list
